src/login/logind-varlink.c

```diff
@@ -251,6 +233,9 @@ static int vl_method_create_session(sd_varlink *link, sd_json_variant *parameter
                         return log_debug_errno(r, "Failed to get peer pidref: %m");
         }
 
+        if (p.pid.fd < 0)
```

Comment:
Then, maybe better to replace `SD_JSON_RELAX` for json_dispatch_pidref() in the above table with `SD_JSON_STRICT` ?

Reply:
Not quite, since we still need the PIDREF_AUTOMATIC semantics enabled via _RELAX...

Reply:
Is it really necessary to support AUTOMATIC thing? When sender wants to send message with all zero `"PID"` field? Why not to just send without the `"PID"` field?
Actually when DBus method is used, we needs to send zero for PID. But with varlink method we can avoid sending meaningless fields.